idea-plugin/src/main/java/com/palantir/gradle/jdks/GradleJdksProjectService.java

```diff
@@ -111,6 +111,7 @@ public void maybeSetupGradleJdks() {
         TasksKt.withBackgroundProgress(
                 project,
                 "Gradle JDK Setup",
+                TaskCancellation.nonCancellable(),
```

Comment:
Previously the user had a button "x" to cancel the background task, but even if they clicked the button nothing would happen until the `./gradle-jdks-setup` script finished running.
<img width="495" alt="Screenshot 2024-09-17 at 12 12 05 PM" src="https://github.com/user-attachments/assets/b27ec437-e709-4aa6-9977-5c070fb257c6">